static/skin/taskbar.js

```diff
@@ -40,65 +41,17 @@ function on_ready() {
         },
         resultItem: {
           element: (item, data) => {
-            item.innerHTML = `${htmlDecode(data.value.label)}`;
+            let searchLink;
+            if (data.value.kind == "path") {
+              searchLink = `${root}/${bookName}/${encodeURI(data.value.path)}`;
+            } else {
+              searchLink = `${root}/search?content=${bookName}&pattern=${data.value.value}`;
+            }
+            item.innerHTML = `<a class="suggest" href="${searchLink}">${htmlDecode(data.value.label)}</a>`;
           }
         }
       }
     );
-
-  /*
-        // cybook hack
-        if (navigator.userAgent.indexOf("bookeen/cybook") != -1) {
-            $("html").addClass("cybook");
-        }
-    
-        if ($(window).width() < 520) {
-            var didScroll;
-            var lastScrollTop = 0;
-            var delta = 5;
-            // on scroll, let the interval function know the user has scrolled
-            $(window).scroll(function (event) {
-                didScroll = true;
-            });
-            // run hasScrolled() and reset didScroll status
-            setInterval(function () {
-                if (didScroll) {
-                    hasScrolled();
-                    didScroll = false;
-                }
-            }, 250);
-            function hasScrolled() {
-                var st = $(this).scrollTop();
-    
-                // Make sure they scroll more than delta
-                if (Math.abs(lastScrollTop - st) <= delta)
-                    return;
-    
-                // If they scrolled down and are past the navbar, add class .nav-up.
-                // This is necessary so you never see what is "behind" the navbar.
-                if (st > lastScrollTop) {
-                    // Scroll Down
-                    $('#kiwixtoolbar').css({ top: '-100%' });
-                } else {
-                    // Scroll Up
-                    $('#kiwixtoolbar').css({ top: '0' });
-                }
-    
-                lastScrollTop = st;
-            }
-        }
-    
-        $('#kiwixsearchbox').on({
-            focus: function () {
-                $('.kiwix_searchform').addClass('full_width');
-                $('label[for="kiwix_button_show_toggle"], .kiwix_button_cont').addClass('searching');
-            },
-            blur: function () {
-                $('.kiwix_searchform').removeClass('full_width');
-                $('label[for="kiwix_button_show_toggle"], .kiwix_button_cont').removeClass('searching');
-            }
-        });
-    };*/
```

Comment:
I have commented this code because I don't know what to do with it (and I was not sure of the purpose of it).
We may remove the cybook hack if it is not relevant anymore (but we have to state that explicitly)
But the code with the scrool and focus/blur is probably useful at a moment.